src/predicates/intersects.jl

```diff
@@ -168,8 +247,18 @@ intersects(m::Multi, c::Chain) = intersects(c, m)
 function minkowskipoint(g₁::Geometry{Dim,T}, g₂::Geometry{Dim,T}, d) where {Dim,T}
   n = Vec{Dim,T}(d[1:Dim])
   v = supportfun(g₁, n) - supportfun(g₂, -n)
-  Point(ntuple(i -> i ≤ Dim ? v[i] : zero(T), max(Dim, 3)))
+  Point(ntuple(i -> i ≤ Dim ? v[i] : zero(T), Dim))
 end
 
 # origin of coordinate system
-minkowskiorigin(Dim, T) = Point(ntuple(i -> zero(T), max(Dim, 3)))
+minkowskiorigin(Dim, T) = Point(ntuple(i -> zero(T), Dim))
+
+# find a vector perpendicular to `v` using vector `d` as some direction hint
+# expect that `perpendicular(v, d) ⋅ d ≥ 0` or, in other words,
+# that the angle between the result vector and `d` is less or equal than 90º
+function perpendicular(v::Vec{2,T}, d::Vec{2,T}) where T
+  a = Vec(v[1], v[2], zero(T))
+  b = Vec(d[1], d[2], zero(T))
+  r = a × b × a
+  Vec(r[1], r[2])
+end
```

Comment:
Very good idea 💯

Reply:
Thank you. I tried to rewrite the function to use only 2D vectors.
It was something like:
```julia
function perpendicular(v::Vec{2,T}, d::Vec{2,T}) where T
  M = @SMatrix T[0 -1; 1 0] # rotation matrix 90º
  M * (sign(det([v d])) * v) # det([v d]) is proportional to sin(angle(v, d))
end
```

It almost worked, but it failed in just one test. So, I didn't use it.

Now, I just need to write some tests.

Reply:
It would be nice to use the function in the 3D case for consistency, even if the implementation differs. We can dispatch on the dimension again to trigger the correct implementation.